### ISSUE TITLE
Rosetta Code: 24 game

### ISSUE THREAD

**Issue by margielewski** [freeCodeCamp/freeCodeCamp#38154]
in Test section
 'solve24("1127") should return a permutation of (1+7)\*(1\*2)'
(1+7)\*(1\*2)===16
i think in the second bracket should be '+' : 
(1+7)\*(1+2)===24

**Comment by ojeytonwilliams** [comment]
@margielewski I just escaped some \*s which were being interpreted as markdown.

**Comment by RandellDawson** [comment]
@ojeytonwilliams I don't think it is a markdown issue.

```
  - text: <code>solve24("1127")</code> should return a permutation of <code>(1+7)*(1*2)</code>
    testString: assert(include(answers[3], solve24(testCases[3])));
```

**Comment by ojeytonwilliams** [comment]
Closed by #38102
